README.md

```diff
@@ -34,8 +34,12 @@ Go to `https://localhost`, and enjoy!
 * [Early Hints support (103 HTTP status code)](docs/early-hints.md)
 * [Real-time](docs/mercure.md)
 * [Configuration](docs/config.md)
-* [Demo app (Symfony) and benchmarks](https://github.com/dunglas/frankenphp-demo)
-* [Building Docker images](docs/docker.md)
 * [Compile from sources](docs/compile.md)
+* [Building Docker images](docs/docker.md)
```

Comment:
I'm just curious because the diff looks weird, but did you intend to move the link down one?